pytorch_lightning/callbacks/finetuning.py

```diff
@@ -83,16 +83,14 @@ def __init__(self):
         self._internal_optimizer_metadata: Dict[int, List[Dict[str, Any]]] = {}
         self._restarting = False
 
-    def on_save_checkpoint(
-        self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", checkpoint: Dict[str, Any]
-    ) -> Dict[int, List[Dict[str, Any]]]:
-        return self._internal_optimizer_metadata
+    def state_dict(self) -> Dict[str, Any]:
```

Comment:
This would make backward checkpoints not reloadable with base-finetuning.

Reply:
added below